modules/monitoring-about-creating-alerting-rules-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
```

Comment:
I think we can remove the last bullet `Optimize alert routing` from `Optimizing alerting for user-defined projects`.

Reply:
Good catch! Removed.